Cargo.toml

```diff
@@ -28,11 +40,12 @@ futures = "0.3.19"
 serde = "1"
 serde_json = "1"
 sha2 = "0.10.1"
-tokio = { version = "1", features = ["io-std", "io-util"] }
 thiserror = "1.0.30"
 url = "2.2.2"
 
 [dev-dependencies]
 import_map = "0.7.0"
 pretty_assertions = "1"
-tokio = { version = "1", features = ["full"] }
+tokio = { version = "1", features = ["macros", "rt"] }
+deno_core = { version = "0.164.0" }
```

Comment:
Adding this makes the eszip repo depend on the deno repo, but the deno repo already depends on eszip. So it creates a circular relationship which could lead to needing to break the circle in the future which is why I ended up needing to remove the examples in order to upgrade swc. Is there a way we could write the examples without deno_core or some solution to this issue?

Reply:
Thanks for the explanation, I get it, so let me try to figure out if there's a good way to not depend on deno_core.

Reply:
I think there are two solutions.
The first one is that we remove the `eszip_load` example, because it's the only example that depends on deno_core and it would be impossible to make it not depend on deno_core considering what this example does.
The second solution is that we create a new repository that will contain the three examples. By doing so, the newly created repository will depend on deno_core (as well as eszip), while we'll be able to remove the dependency on deno_core from the eszip repo, getting rid of circular dependency.

I will go with the first option for the moment, because what I'd like to use for my current use case is eszip_viewer only.

Reply:
I think the first option sounds good. A second repository would probably get out of date.

Maybe we could also maybe see about unpinning pinned dependencies in deno_core and only pin them in the CLI crate. I'm not sure if that would help though.